### ISSUE TITLE
So, is this repo a demo on int8 or int32?

### ISSUE THREAD

**Issue by Jonxjdong** [RAYKALI/simple-int8-pytorch-implement#1]

**Comment by RAYKALI** [comment]
Int8. All weights range from -128 to 127   
This is just an example of q processes，In the actual q process, torch.utils.cpp_extension and nvidia-DP4A need to be used together.
Also,pytorch new characteristics 
https://pytorch.org/docs/stable/quantization.html 
maybe helpful.

**Comment by Jonxjdong** [comment]
well， thank you very much

**Comment by Jonxjdong** [comment]
Thanks. 
Btw, are the images in ./dataset/q/ just randomly selected from downloaded dataset?

**Comment by RAYKALI** [comment]
The official plan is: 500(enough) samples are selected manually from the test set, all kinds of samples should be class balanced and contain class's distinctive features

**Comment by RAYKALI** [comment]
500 for voc(20classes)
